### DISCUSSION TITLE
Update "Documentation" section of README.MD

### DISCUSSION THREAD

**Discussion by pawelWritesCode** [cucumber/godog#467] [Ideas]
Hi, 

I'm working with godog for some time and I came up with idea of writting skeleton repo that sets basic godog structures and  develop it for HTTP API testing purposes.

Result of my work is here: https://github.com/pawelWritesCode/godog-http-api

My goal is to make user's life easier. That's why I emphasize ease of setup, properly documenting code and provide maintanable architecture. When I switched from PHP(behat) to GO(godog) I struggled with finding good library with gherkin steps for testing HTTP API. Now, I would like to contribute to community and share my work so others may benefit from my experience - I hope this setup may be good guide for anyone who wants to start using godog and struggle with finding medium size egzample. That's why I need you, to tell me whether you find this repository useful and is worth to mention in README.MD? I would be glad to hear any feedback from you.

Mostly, I would like to know whether you find it properly developed in terms of godog primitives/standards and Do I have green light to create pull request to README.MD with link to this repo?

**Comment by vearutop** [comment]
Hi, this is interesting and seems useful, however somewhat similar proposal was previously rejected here https://github.com/cucumber/godog/issues/410 and the rationale still applies.

**Reply by pawelWritesCode** [reply]
Ok, thanks for reply
